### DISCUSSION TITLE
Driver es1370

### DISCUSSION THREAD

**Discussion by mxlgv** [klange/toaruos#279] [Q&A] [answered]
Hello! Tell me please, did you write `es1371` driver for real hardware? I just discovered that qemu has `es1370` emulation. I wanted to write this driver for Qemu for a long time, it seems to be not much different from `es1371`, at least I determined it from the NetBSD and Minix3 sources.

**Comment by klange** [comment, marked as answer]
https://github.com/klange/toaruos/blob/master/modules/es1371.c

This driver was written for VMware Workstation quite some time ago.

**Reply by klange** [reply]
The main difference in the es1371 is AC97 compatibility.
